<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>popup</class>
 <widget class="QMainWindow" name="popup">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>187</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>619</width>
    <height>413</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Canvas Project</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../AppData/Downloads/canvascircle_g0O_icon.ico</normaloff>../../AppData/Downloads/canvascircle_g0O_icon.ico</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">
background: qlineargradient(x1: 0, y1: 0, x2: 1, y2: 1, stop: 0 rgba(50,13,13, 1), stop: 0.1 rgba(13,13,13, 1), stop: 1 rgba(30,13,13, 1));
</string>
  </property>
  <property name="iconSize">
   <size>
    <width>24</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="logo">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>351</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/images/canvasproject_small.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid rgb(195, 195, 195);
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}</string>
    </property>
    <property name="text">
     <string>Sähköposti:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid rgb(195, 195, 195);
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}</string>
    </property>
    <property name="text">
     <string>Salasana:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	border: 1px solid rgb(195, 195, 195);
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}</string>
    </property>
    <property name="text">
     <string>Lisää Canvas tunnukset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: 1px solid rgb(195, 195, 195);
	border-radius: 1px;	
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}
QPushButton:hover {
	border: 2px solid rgb(195, 195, 195);
	background-color: rgb(55,55,55);
}
QPushButton:pressed {	
	border: 2px solid rgb(230, 230, 230);
	background-color: rgb(60,60,60);
}

QPushButton:disabled {	
	border-radius: 10px;	
	border: 2px solid rgb(66, 66, 66);
	background-color:rgb(115,115,115);
	color:rgb(255,255,255);
	font: 8pt &quot;MS Shell Dlg 2&quot;;
	text-decoration: line-through;
	font-size:11pt;


}</string>
    </property>
    <property name="text">
     <string>Lisää tunnukset</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="username">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid rgb(195, 195, 195);
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="password">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>90</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
	border: 1px solid rgb(195, 195, 195);
	color:rgb(255,255,255);
	background-color: rgb(45,45,45);
	font-size:11pt;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../AppData/Local/Temp/assets.qrc"/>
 </resources>
 <connections/>
</ui>
